composer.json

```diff
@@ -19,13 +19,13 @@
         }
     ],
     "require": {
-        "php" : ">=7.0.10",
+        "php": "^7.2",
         "league/csv": "~9.0",
         "mustangostang/spyc": "~0.6",
-        "illuminate/support": "~5.5"
+        "illuminate/support": "^6.0"

```

Comment:
Is this upgrade necessary? I'd just try to push it up with a range here to see if you can support both versions.